### PR TITLE
fix #45 - update README for Firefox & Chrome

### LINKED ISSUE #45: Streamline install experience for developers via README
Before we present to TWD, I want to go thru our README install instructions a number of times and make it very smooth.

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  * `gulp dev`
+
+## To use the Chrome Extension
+1. Follow the [Unpacked Chrome Extensions
+   docs](http://developer.chrome.com/extensions/getstarted.html#unpacked) and load the `chrome` directory
+
+## To use the Firefox Add-on
+1. `cd firefox`
+2. Run Firefox via jpm:
+  * `../node_modules/.bin jpm run`
```
Comment (jdungan) [inline README.md]: This doesn't work.  Needs to be `../node_modules/.bin/jpm run

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Note: You can [use `-p <PROFILE>` or `--profile
+<PROFILE>`](https://github.com/mozilla/jpm#usage)  to use your specific
+[Firefox
+profile](https://support.mozilla.org/en-US/kb/profiles-where-firefox-stores-user-data).
+
+## Publish (Chrome-only)
```
Comment (jdungan) [inline README.md]: I forgot I renamed the task to publish-chrome.  Plus it doesn't work.  Need to file an issue on it.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+The codesy.io widget is a Chrome Extension and Firefox Addon the adds codesy.io interface to open-source bugs you visit.
+
+
+## To work locally
+
+1. Clone this repo:
+  * `git clone https://github.com/codesy/widgets.git`
+2. Install requirements:
+  * `cd widgets`
+  * `npm install`
```
Comment (jdungan) [inline README.md]: Do we need to add 'gulp' to package.json so the it is installed?

**Comment by groovecoder** [comment]
Updated.

**Pull request merged**